src/Interpreter.js

```diff
@@ -112,7 +114,7 @@ export default class Interpreter {
     callCommandHandlers(handlers: Array<CommandHandler>): Promise<any> {
         const promises = [];
         for (const handler of handlers) {
```

Comment:
Grab the `stepTimeMs` before entering the handler loop:

    const stepTimeMs = this.stepTimeMs;

And use this value in the loop to ensure that every handler for this step is given the same value.